### PR TITLE
Helm Message Parser

### PR DESCRIPTION
**Pull request by KennethSkylight**
# Helm Message Parser

Adds the helm chart for message parser.

### CODE REVIEW

File: charts/message-parser/Chart.yaml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+apiVersion: v2
+name: message-parser
+description: A Helm chart for Kubernetes
+type: application
+version: 0.1.0
+appVersion: "1.16.0"
```
Comment (emmastephenson) [inline charts/message-parser/Chart.yaml:6]: I think our PHDI version is only on `1.0.9` - should we change `appVersion` accordingly?

File: charts/message-parser/templates/deployment.yaml [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  template:
+    metadata:
+      annotations:
+      labels:
+        app: message-parser-app
+    spec:
+      imagePullSecrets:
+      containers:
+        - name: {{ .Chart.Name }}
+          image: ghcr.io/cdcgov/phdi/message-parser:{{ .Values.image.tag }}
```
Comment (emmastephenson) [inline charts/message-parser/templates/deployment.yaml:21]: We may want a default value here - maybe `latest` if the image tag isn't specified?

**Review by emmastephenson — COMMENTED** [review]
Looks good! Just some minor comments.

File: charts/message-parser/templates/deployment.yaml (hunk 2 above)
Comment (KennethSkylight) [reply charts/message-parser/templates/deployment.yaml]: changed

File: charts/message-parser/Chart.yaml (hunk 1 above)
Comment (KennethSkylight) [reply charts/message-parser/Chart.yaml:6]: changed

File: charts/message-parser/templates/deployment.yaml [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+  template:
+    metadata:
+      annotations:
+      labels:
+        app: message-parser-app
+    spec:
+      imagePullSecrets:
+      containers:
+        - name: {{ .Chart.Name }}
+          image: ghcr.io/cdcgov/phdi/message-parser:{{ default "latest" .Values.image.tag }}
```
Comment (emmastephenson) [inline charts/message-parser/templates/deployment.yaml:21]: I think you can set the default dynamically, with

`{{ .Values.image.tag | "latest" }}`

I'm not sure the template will be able to parse it as written.

**Review by emmastephenson — CHANGES_REQUESTED** [review]

File: charts/message-parser/templates/deployment.yaml (hunk 3 above)
Comment (KennethSkylight) [reply charts/message-parser/templates/deployment.yaml]: It did when I ran it, but it looked really weird...

Comment (KennethSkylight) [reply charts/message-parser/templates/deployment.yaml]: changed

**Pull request merged**